README.md

```diff
@@ -84,3 +84,4 @@ We welcome contributions to `xdggs`! Please follow these steps to get involved:
 ## Acknowledgments
 
 This project was inspired by the increasing need for scalable geospatial data analysis using DGGS and is built upon the robust ecosystem of Xarray.
```

Comment:
To make the acknowledgements read better, we should not have multiple sentences starting with "this project". In particular, I'd actually remove this sentence since it does not add a lot of value in addition to the project description (but I'd be happy to be outvoted on this one):
```suggestion
```

Reply:
I agree to take that out, we can move this line to somewhere else. (more in the documentation)